src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const erorr = {};
+    erorr.errors = [];
+
+    const [originalText, query] = req.url.slice(1).split('?');
+    const params = new URLSearchParams(query);
+    const targetCase = params.get('toCase');
+
+    if (!originalText) {
+      erorr.errors.push({
+        message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`
+      });
+    }
+
+
+    if (!targetCase) {
+      erorr.errors.push({
+        message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`
+      });
+    }
+
+    if (targetCase) {
+      try {
+        convertToCase(originalText, targetCase);
+      } catch (err) {
+        erorr.errors.push({
+          message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.'
+        });
+      }
+    }
+
+    if (erorr.errors.length > 0) {
+      res.statusCode = 400;
+      res.statusMessage = 'Bad request';
+      res.end(JSON.stringify(erorr));
+    } else {
```

Comment:
i think if you remove else it would still work the same (but then you need to add `return` to your if block, so everything else after if wont be executed)

Reply:
Good point!